src/vuefire.js

```diff
@@ -61,11 +85,7 @@ function bind (vm, key, source) {
   if (!isObject(source)) {
     throw new Error('VueFire: invalid Firebase binding source.')
   }
-  // get the original ref for possible queries
-  var ref = source
-  if (typeof source.ref === 'function') {
```

Comment:
Ahh I'm mistaken. Sorry about that. Your code in the `_getRef()` method handle the case I was thinking about.